src/Carbon/CarbonInterval.php

```diff
@@ -1600,6 +1606,12 @@ public function forHumans($syntax = null, $short = false, $parts = -1, $options
             if (count($interval) >= $parts) {
                 break;
             }
+
+            // break the loop after we have reached the minimum unit
+            if (in_array($minimumUnit, [$diffIntervalData['unit'], $diffIntervalData['unitShort']])) {
+                $fallbackUnit = [$diffIntervalData['unit'], $diffIntervalData['unitShort']];
+                break;
```

Comment:
A blank line before statements is required by php-cs code style:
https://travis-ci.org/github/briannesbitt/Carbon/jobs/672454794#L444